### PR TITLE
feat: Change some UI of the menu members card when using a mobile - MEED-1518 - Meeds-io/meeds#549

### PR DESCRIPTION
**Pull request by MayTekayaa**
This fix will change some UI of the menu members card when using a mobile.

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue [hunk 1]
```diff
@@ -61,6 +62,80 @@
         <span class="d-flex uiIconPLFFont uiIconGroup"></span>
       </v-btn>
       <v-spacer />
+      <div v-if="isMobile && !isSameUser">
+        <v-icon 
+          size="14" 
+          class="my-3" 
+          @click="openBottomMenu">
+          fa-ellipsis-v
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue]:
```suggestion
          fas fa-ellipsis-v
```

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue [hunk 2]
```diff
@@ -61,6 +62,80 @@
...
+        </v-icon>
+        <v-bottom-sheet v-model="bottomMenu" class="pa-0">
+          <v-sheet class="text-center" height="100px">
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue]:
```suggestion
          <v-sheet class="text-center" height="100">
```

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue [hunk 3]
```diff
@@ -61,6 +62,80 @@
...
+            <v-list>
+              <v-list-item 
+                v-if="user.relationshipStatus === 'CONFIRMED'"
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue]: To add in computed, please!

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue [hunk 4]
```diff
@@ -61,6 +62,80 @@
...
+                @click="disconnect">
+                <v-list-item-title class="align-center d-flex">
+                  <v-icon class="mx-4" size="16">fa-minus-circle</v-icon>
+                  <span class="mx-2">
+                    {{ $t('peopleList.button.disconnect') }}
+                  </span>
+                </v-list-item-title>
+              </v-list-item>
+              <v-list-item 
+                v-else-if="user.relationshipStatus === 'INCOMING'">
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue]: To add in computed, please!

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue [hunk 5]
```diff
@@ -61,6 +62,80 @@
...
+                  <span class="mx-2">
+                    {{ $t('peopleList.button.cancelRequest') }}
+                  </span>
+                </v-list-item-title>
+              </v-list-item>
+              <v-list-item 
+                v-else
+                @click="connect">
+                <v-list-item-title class="align-center d-flex">
+                  <v-icon class="mx-4" size="16">fa-plus-circle</v-icon>
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue]:
```suggestion
                  <v-icon class="mx-4" size="16">fas fa-plus-circle</v-icon>
```

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue (hunk 1 above)
Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue (hunk 2 above)
Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue (hunk 3 above)
Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue (hunk 5 above)
Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue]: done!

**Review by AzmiTouil — APPROVED** [review]

**Pull request merged**